Breeze/src/Breeze.Daemon/Properties/launchSettings.json

```diff
@@ -6,7 +6,7 @@
     },
     "Breeze.Daemon TestNet": {
       "commandName": "Project",
-      "commandLineArgs": "-testnet -tumbler-uri=http://localhost:5050 light -debug=all -loglevel=debug"
+      "commandLineArgs": "light -testnet"
```

Comment:
Why remove the debug arguments?

Reply:
I've never used them, nor seen anyone using them and they're not documented. What do they do? Should I keep them there?

Reply:
Yes please keep them there, they're use to set the level of debugging we wish to have while running this project.
You can remove the tumbler thingy though, that's a thing from the past.